packages/account/src/Sections/Security/ConnectedApps/connected-apps-article.jsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { localize, Localize } from '@deriv/translations';
+import AccountArticle from 'Components/article';
+
+const openAPIManagingWebsite = () => {
+    window.open(
+        'https://community.deriv.com/t/api-tokens-managing-access-on-third-party-applications-and-mobile-apps/29159',
+        '_blank'
+    );
+};
+
+const ConnectedAppsArticle = () => (
+    <AccountArticle
+        className='connected-apps__article'
+        title={localize('Learn more about APIs')}
+        descriptions={[
+            <Localize
+                key={0}
+                i18n_default_text='Go to our Deriv community and learn about APIs, API tokens, ways to use Deriv APIs, and more.'
+            />,
+        ]}
+        onClickLearnMore={openAPIManagingWebsite}
```

Comment:
There is a mismatch between the design and code here, in the design we can see `Take me there` button which is `Learn more` in the code, could you clarify on this?

Reply:
Hi @mahdiyeh-fs , the existing design for the button in the component is `Learn more` I have asked the PM to check with the design again now waiting for the updates 🙏

Reply:
@suisin-deriv did you get the confirmation for `Learn more` button ?

Reply:
@mahdiyeh-fs yes the design is updated 🙏